conf/defaults.ini

```diff
@@ -397,6 +397,7 @@ global_session = -1
 [alerting]
 # Makes it possible to turn off alert rule execution.
 execute_alerts = true
+show_alerting_ui = true
```

Comment:
In that case, it might get confusing for some folks. "enabled" would mean that alerting itself is enabled or disabled, but we are just hiding the UI. However, alerts would still be executed(if there are any already setup) if "execute_alerts" is true.